templates/car_input.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <h1>Welcome to the Automotive Solution Tool!</h1>
```

Comment:
We can move this to the body, our header will handle the `<title> CHOOSE </title>`

We can also include a stylesheet link to bootstrap for generic templating